doc/reference/tx-metadata.md

```diff
@@ -1,16 +1,75 @@
 # Transaction Metadata
 
-Transaction metadata gives us the ability to put arbitrarily structured data onto the chain. The metadata is not involved in transaction validation and is intended to be consumed by applications (i.e is application specific). It is only metadata from the point of view that it is carried along with transactions. The metadata value is kept outside the transaction body but the metadata hash is in the transaction body allowing for integrity checking and authentication of the metadata.
+Transaction metadata (tx metadata) contains details about a specific transaction, including sender and receiver IDs, transaction comments, and tags. Adding metadata to transactions  provides transaction information by adding arbitrarily structured data onto the chain and is a useful feature in Cardano Shelley. Tx metadata is stored on-chain and is carried along with each transaction. A factor in its design is that the on-chain metadata is not stored in the ledger state and does not influence transaction validation, thereby not compromising ledger performance.
 
-## Metadata Schemas/Mappings/Formats
+Tx metadata is a valuable feature for developers who build applications and process transactions, and for application end users. Developers embed metadata directly and submit a valid transaction with accompanying details. End users don’t interact with the tx metadata directly, but can view transaction-specific metadata using the Cardano Explorer.
 
-On the chain, within transactions, the metadata is encoded as CBOR. When
-building transactions the metadata can be supplied as pre-encoded CBOR.
-Alternatively, metadata can be supplied as JSON which will be converted
-into the internal format. We provide two different mappings between tx
-metadata and JSON, useful for different purposes.
-In both schemes, the top level JSON is an object indexed by integers which
-are mapped to JSON values.
+For example, tx metadata can be used to:
+
++ certify ownership and exchange of assets specifying asset owners in different time periods, transfer details, or asset value at the time of transfer
++ certify documents and signatures, by using a public hash that proves the document’s existence 
+
+Transactions can contain metadata whose hash is part of the body of the transaction. This hash is read by all transaction signatures. The metadata value is kept outside the transaction to allow checking its integrity and authentication.
```

Comment:
Instead of:
> This hash is read by all transaction signatures. The metadata value is kept outside the transaction to allow checking its integrity and authentication.

I would say something like:
> Because the metadata hash is in the tx body, this allows for integrity checking and authentication of the metadata